### DISCUSSION TITLE
[ FF7 ] Oops! Something very bad happened.   Wrote emergency save to save/crash.ff7 dir.

### DISCUSSION THREAD

**Discussion by KhalDrogoSings** [julianxhokaxhiu/FFNx#181] [General]
Using 7th  Heaven v2.2.3.522 here 

I got an error saying I should upload 2 files here. Any help?

The game crashes when I'm not in the world map.

[FF7.zip](https://github.com/julianxhokaxhiu/FFNx/files/5398028/FF7.zip)

**Comment by julianxhokaxhiu** [comment]
Thanks for having me approached. If you use 7th you need to approach the 7th related support channels. Cheers!
